test/modules/post/test/file.rb

```diff
@@ -212,8 +212,9 @@ def test_file
 
   def test_binary_files
     # binary_data = ::File.read("/bin/ls")
-    binary_data = ::File.read('/bin/echo')
+    # binary_data = ::File.read('/bin/echo')
     # binary_data = "\xff\x00\xff\xfe\xff\`$(echo blha)\`"
+    binary_data = ((0..255).to_a * 500).shuffle.pack("c*")
```

Comment:
Should this not be packed as `C` aka an unsigned character? Don't think we need signed data here?

Reply:
That's technically accurate but in Ruby it doesn't appear to matter.

```
[1] pry(main)> [255].pack('c*')
=> "\xFF"
```